src/locales/default.json

```diff
@@ -264,10 +275,26 @@
     "tradeLiquidity": "Trade liquidity",
     "transactionPending": "No transactions pending | {n} transaction pending | {n} transactions pending",
     "transactionDeadline": "Transaction deadline",
-    "transactionDeadlineTooltip": "Your swap expires and will not execute if it is pending for longer than the selected duration. Only executed swaps incur fees.",
+    "transactionDeadlineTooltip": "Your swap will expire and not execute if it is pending for more than the selected duration. Only executed swaps incur fees for trades between ERC-20 tokens.",
     "unavailableOnNetwork": "Sorry! Balancer is not available on this network",
     "unavailableOnNetworkWithName": "Sorry! Balancer is not available on {0}",
     "liquidityProviderCopy": "People who provide liquidity to eligible investment pools or trade on eligible token pairs receive weekly $BAL distributions as incentives. $BAL token holders own the Balancer Protocol and can help shape its future by voting on governance proposals.",
+    "gnosisErrors": {
+        "lowAmount": {
+            "header": "Your trade amount is too low",
+            "body": "The fees for this transaction exceeds the amount you want to trade."
+        },
+        "lowBalance": {
+            "header": "Insufficient {0} balance",
+            "body": "The fees and potential slippage (up to {2}) set for this trade exceeds your {0} balance. You need at least {1} {0} to proceed."
+        }
+    },
+    "gnosisWarnings": {
+        "highFees": {
+            "header": "The fees on this transaction exceed 20% of the trade amount.",
+            "body": "You can still proceed with the trade but a high percentage of it will be consumed by trade fees."
+        }
+    },
```

Comment:
I think this pattern of keys is much more manageable and is not dependent on the actual content.